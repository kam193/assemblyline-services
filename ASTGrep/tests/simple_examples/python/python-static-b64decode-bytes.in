import base64
from base64 import b64decode

base64.b64decode(b"YWJj")
b64decode(b'YWJj')

binascii.a2b_base64(b'YWJj')
