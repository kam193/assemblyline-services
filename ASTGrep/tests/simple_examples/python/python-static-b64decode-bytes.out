import base64
from base64 import b64decode

b'abc'
b'abc'

b'abc'
